### PR TITLE
reset value of input field of SearchWidget

### LINKED ISSUE #4028: Input value of Serach field remains even after hitting submit button
## Issue
When the input is given inside the search field the value remains there even if we change our routes or do editing etc. which I think does not look so good.
This video illustrates the above issue:

https://user-images.githubusercontent.com/99530996/205440822-41bf8e6b-dea0-4112-99e3-5b874f29f9cf.mp4

## Approach

1. In `plone/volto/src/components/theme/SearchWidget/SearchWidget.jsx` we can modify the handler `onSubmit()` to reset the value of the input field of the search widget.
OR
2. As proposed by tiberiuichim#7386 on discord we can replace the input field with a button that takes you to the search page just like in bbc.com.

https://user-images.githubusercontent.com/99530996/205442462-cb27c76c-0d67-48fe-aecc-e159f4c1b076.mp4

Feedback Welcome.

### PR DESCRIPTION
**Pull request by MAX-786**
fixes #4028 

## Approach
reset the value of text using setState() :
```js
  onSubmit(event) {
    const path =
      this.props.pathname?.length > 0
        ? `&path=${encodeURIComponent(this.props.pathname)}`
        : '';
    this.props.history.push(
      `/search?SearchableText=${encodeURIComponent(this.state.text)}${path}`,
    );
 // *** reset value of input field
    this.setState({
      text: '',
    });
 // ***
    event.preventDefault();
  }

```
## Demo:

https://user-images.githubusercontent.com/99530996/206011822-4c33fdb8-2547-4456-a68f-c6c373c07c95.mp4

### OS: Any

Feedback Welcome.

### CODE REVIEW

**Review by davisagli — COMMENTED** [review]
@MAX-786 This looks like a good fix to me. Thank you! Please add an entry for the changelog by creating the file news/4028.bugfix with a one-line description of the fix, and your github username in [brackets]

**Comment by MAX-786** [comment]
It's done! 😊

File: news/4028.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+reset value of search field after submit. [@MAX-786]
```
Comment (davisagli) [inline news/4028.bugfix:1]:
```suggestion
Reset value of search field after submit. [@MAX-786]
```

**Comment by davisagli** [comment]
@MAX-786 Since your PR was submitted from a fork (MAX-786:searchWidget), it did not trigger the automated tests. Could you please push your branch to this repository (https://github.com/plone/volto) instead of your fork, and open a new PR? Then the tests should run. You should already have access to create the branch since you signed the contributor agreement.

**Comment by MAX-786** [comment]
should I close this one then?

**Comment by davisagli** [comment]
@MAX-786 yeah, unfortunately there isn't a way to switch the existing PR to come from a different branch.

**Comment by MAX-786** [comment]
Ah!, 😣I'll remember next time. Thanks!

**Comment by davisagli** [comment]
@wesleybl It's on my list of things to research, but I haven't gotten to it yet. So far I've been just manually creating a PR that includes the same change as dependabot.

**Pull request closed without merging**